internal/commands/user/inputs.go

```diff
@@ -0,0 +1,120 @@
+package user
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+const (
+	userDataEmail = "email"
+	userDataName  = "name"
+)
+
+// usersInputs are the filtering inputs for a user command
+type usersInputs struct {
+	State         realm.UserState
+	ProviderTypes []string
+	Status        statusType
+	Users         []string
+}
+
+// ResolveUsers will use the provided Realm client to resolve the users specified by the realm.App through inputs
+func (i *usersInputs) ResolveUsers(ui terminal.UI, client realm.Client, app realm.App) ([]realm.User, error) {
+	if len(i.Users) > 0 {
```

Comment:
where did we land on this filter logic being similar to that of the `realm.Client`'s own `FindUsers` function?  I imagined we would have been able to simply convert our input fields to a `realm.UserFilter` here and call it day?

seems like we could fetch all the users here up top, and then if `len(i.Users) > 0` we can just return that list, otherwise we can prompt the user with a multiselect

Reply:
addressed